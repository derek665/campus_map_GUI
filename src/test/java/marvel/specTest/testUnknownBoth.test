# test correct output when both characters are unknown

LoadGraph testBoth soccer.tsv

FindPath testBoth A B