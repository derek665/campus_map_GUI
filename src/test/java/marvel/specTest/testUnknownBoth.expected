# test correct output when both characters are unknown

loaded graph testBoth

unknown character A
unknown character B